tests/compute/test_heterograph.py

```diff
@@ -1944,7 +1944,7 @@ def test_reverse(index_dtype):
         ('user', 'follows', 'user'): ([0, 1, 2, 4, 3 ,1, 3], [1, 2, 3, 2, 0, 0, 1]),
     }, index_dtype=index_dtype)
     gidx = g._graph
-    r_gidx = gidx.reverse()
+    r_gidx = gidx.reverse(gidx.metagraph)
```

Comment:
I think this file is meant to test DGLHeteroGraph, not HeteroGraphIndex. Since we now moved the implementation to transform, to add tests in test_transform.py should be sufficient.

Reply:
This is more like the unitest for single graphidx object.
The test here also check the graph format.

For tests in to_transform, the tests are more like API test.